ToDo/src/css/todo/index.css

```diff
@@ -0,0 +1,26 @@
+/* GENERAL STYLES */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(195, 142, 166);
+}
+
+/* CONTAINER */
+.todo-container {
+  background-color: rgb(250, 250, 250);
+  border-radius: 4px;
+  width: 80vw;
+  min-height: 240px;
+  max-height: 80vh;
+  border: 1px solid white;
+  box-shadow: -2px 2px 2px #999;
+  margin: 10vw 8vw 6vw 5vw;
+}
+
+h1 {
+  padding: 20px 50px 5px 5px;
```

Comment:
add padding to todo-container insted of h1 padding: 
.todo-container {
  ...
  padding: 20px;
}

Reply:
changed